File: Set Mismatch/Set Mismatch.py
nums = [1,2,2,4]

def findErrorNums(nums):
    my_arr = [0] * (len(nums) + 1)
    check = 0
    err = 0

    for num in nums:
        my_arr[num] += 1

    for j in range(1, len(my_arr)):
        if my_arr[j] == 2:
            check = j
        if my_arr[j] == 0:
            err = j

    return [check, err]

print(findErrorNums(nums))

File: Set Mismatch/Set Mismatch.js
let nums = [1, 2, 2, 4];

var findErrorNums = function(nums) {
    let myArr = Array(nums.length + 1).fill(0);
    let check = 0;
    let err = 0;

    for (let i = 0; i < nums.length; i++) {
        myArr[nums[i]] += 1;
    }

    for (let j = 1; j < myArr.length; j++) {
        if (myArr[j] === 2) {
            check = j;
        }
        if (myArr[j] === 0) {
            err = j;
        }
    }

    return [check, err];
};

console.log(findErrorNums(nums)); // Output: [2, 3]